frontend/src/hooks/useActions.ts

```diff
@@ -180,65 +159,17 @@ const useActions = (registerHotkeys = false) => {
         }
-        let isInitialStateUpdated = false
-        if (pasteData.projectType !== project.projectType) {
-          showWarning(`Error: you cannot paste elements from a ${pasteData.projectType} project into a ${project.projectType} project.`)
-          return
-        }
-        const isNewProject = pasteData.projectSource !== project._id
-        // Track which transitions have been updated with new state ids
-        const newTransitions = structuredClone(pasteData.transitions)
-        newTransitions.forEach(transition => {
-          transition.from = null
-          transition.to = null
-        })
-        // Add and select states, comments, and transitions
-        pasteData.states.forEach(state => {
-          // TODO: ensure position isn't out of window
-          state.x += PASTE_POSITION_OFFSET
-          state.y += PASTE_POSITION_OFFSET
-          const newId = createState(state)
-          // Update transitions to new state id
-          pasteData.transitions.forEach((transition, i) => {
-            if (transition.from === state.id && newTransitions[i].from === null) {
-              newTransitions[i].from = newId
-            }
-            if (transition.to === state.id && newTransitions[i].to === null) {
-              newTransitions[i].to = newId
-            }
-          })
-          // Update initial state id if applicable
-          if (pasteData.initialStateId === state.id && !isInitialStateUpdated) {
-            pasteData.initialStateId = newId
-            isInitialStateUpdated = true
-          }
-          state.id = newId
-        })
-        // Error if trying to paste transition without its to and from states
-        if (newTransitions.find(transition => transition.from === null || transition.to === null)) {
-          showWarning('Sorry, there was an error while pasting')
-          removeStates(pasteData.states.map(state => state.id))
-          return
-        }
-        pasteData.transitions = newTransitions
-        selectStates(pasteData.states.map(state => state.id))
-        pasteData.comments.forEach(comment => {
-          // TODO: ensure position isn't out of window
-          comment.x += PASTE_POSITION_OFFSET
-          comment.y += PASTE_POSITION_OFFSET
-          const newId = createComment(comment)
-          comment.id = newId
-        })
-        selectComments(pasteData.comments.map(comment => comment.id))
-        pasteData.transitions.forEach(transition => {
-          const newId = createTransition(transition)
-          transition.id = newId
-        })
-        selectTransitions(pasteData.transitions.map(transition => transition.id))
-        if (isNewProject && pasteData.initialStateId !== null && project.initialState === null) {
-          setStateInitial(pasteData.initialStateId)
+        // createBatch(pasteData, project)
+        const insertResponse = insertGroup(pasteData)
+        // This will be better in TS with enum
+        if (insertResponse.type === 2) {
```

Comment:
```suggestion
        if (insertResponse.type === InsertGroupResponseType.SUCCESS) {
```
Just so we aren't using a magical number